test/helpers.go

```diff
@@ -324,10 +325,11 @@ var ComplexSummaryMock = model.Summary{
 	SeveritySummary: model.SeveritySummary{
 		ScanID: "console",
 		SeverityCounters: map[model.Severity]int{
-			model.SeverityInfo:   0,
-			model.SeverityLow:    0,
-			model.SeverityMedium: 1,
-			model.SeverityHigh:   4,
+			model.SeverityInfo:     0,
+			model.SeverityLow:      0,
+			model.SeverityMedium:   1,
+			model.SeverityHigh:     2,
```

Comment:
Why drop the number of severities for HIGH Results? 
Let's change the helpers to use the critical severity. Then unit tests will also test values other than 0 for CRITICAL.

Reply:
It seems there was a typo in the expected results, resulting in incorrect display of severities, although they were actually at 4 High. 
Pull Request [#6866](https://github.com/Checkmarx/kics/pull/6866) was focused on increasing critical severity tests for all report formats. I'm considering adding more tests specifically to the existing unit tests, but restricted to CLI. Once the this commit is incorporated, I plan to merge both to ensure optimal unit test coverage for this new feature. In my opinion, having more tests is always preferable.

Reply:
Changes are live in the last commit. Added critical severity testing to the pre-existent unit tests.